scripts/stac/imagery/create_stac.py

```diff
@@ -82,6 +87,15 @@ def create_collection(
     return collection
 
 
+def get_providers(licensors: list[str], producers: list[str]) -> list[Provider]:
+    providers: list[Provider] = []
+    for producer_name in producers:
+        providers.append({"name": producer_name, "roles": [ProviderRole.PRODUCER]})
+    for licensor_name in licensors:
+        providers.append({"name": licensor_name, "roles": [ProviderRole.LICENSOR]})
+    return providers
```

Comment:
Is this the desired output for `get_providers(["Prov1", "Prov2"], ["Prov2"])`?
`[{'name': 'Prov2', 'roles': ['producer']}, {'name': 'Prov1', 'roles': ['licensor']}, {'name': 'Prov2', 'roles': ['licensor']}]`
Or would we be expecting something like this instead:
`[{'name': 'Prov2', 'roles': ['producer', 'licensor']}, {'name': 'Prov1', 'roles': ['licensor']}]`
?

Reply:
Good point, but I think the current code is identical to the original. I just had to reduce the number of local variables for pylint.